draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -843,9 +843,11 @@ this parameter is not safe to accept over an untrusted channel.
 There is no SvcParamKey corresponding to the Alt-Svc "ma" (max age) parameter.
 Instead, server operators SHOULD encode the expiration time in the DNS TTL.
 
-Some DNS caching systems incorrectly extend the lifetime of DNS
-records beyond the stated TTL.  Server operators MUST NOT rely on
-HTTPSSVC records expiring on time, and MAY shorten the TTL to compensate.
+The appropriate TTL value will typically be similar to the "ma" value
+used for Alt-Svc, but may vary depending on the desired efficiency and
+agility.  Some DNS caching systems incorrectly extend the lifetime of DNS
+records beyond the stated TTL, so server operators MUST NOT rely on
```

Comment:
"MUST NOT" seems wrong here.  Maybe this can be non-normative operational guidance.

Reply:
Agreed, especially as ESNI key rotations need to assume that clients will at least eventually re-resolve and not extend the lifetime indefinitely.  Perhaps "[...], so server operators should assume that clients will continue to use records for some amount of time past their expiration."   ?

Reply:
I've adjusted the advice here.  PTAL.